packages/network-controller/src/NetworkController.ts

```diff
@@ -877,6 +893,29 @@ export class NetworkController extends BaseControllerV2<
   async destroy() {
     await this.#blockTrackerProxy?.destroy();
   }
+
+  /**
+   * Returns data that will be preserved in a wallet backup.
+   *
+   * @returns The data that will be backed up.
+   */
+  captureBackup(): Backup {
```

Comment:
What is the purpose of this method? The state is public, so that can already be accessed just as it was before.

Reply:
Sure. If we can restore from a backup, then it seems like 1) we ought to have a definition of what the controller thinks is the data that will be included in the backup and 2) we ought to have a way to produce that data in the first place. As it stands, the extension is doing of these things, but that doesn't seem right to me. I think it ought to be the responsibility of the controller. What do you think?

Reply:
If we did want that, the state metadata might be a better place for it. That's where we store all other information about what state should be used for. That seems like it could wait until later though.

It might make sense for the controllers to track what data is backed up or not, but I'm not sure because we don't yet have clear criteria for that. The backup feature was hastily implemented to cover the features previously provided by 3Box; it's not clear what we want from that feature as a whole. We might be able to just include all persisted state, removing the necessity for any additional metadata.

Reply:
I see. Yeah, it seems like this feature is pretty bare-bones — I had a similar feeling that we ought to include more state — so I didn't know if it was a legacy feature or not. But it makes sense that it was a replacement for 3Box.

What do you feel like we ought to do? As you say, we technically don't need this method. So we could remove the `Backup` type and just add `loadBackup`, which would take in `{ networkConfigurations: NetworkConfigurations }`. Oh, and I guess it should also merge in the state instead of replacing it (to mimic `updateState` from BaseController).

Reply:
Agreed - that sounds reasonable to me

Reply:
Updated and squashed PR to only include `loadBackup`.